modules/auth/ldap/ldap.go

```diff
@@ -149,18 +158,39 @@ func bindUser(l *ldap.Conn, userDN, passwd string) error {
 	return err
 }
 
+func checkAdmin(l *ldap.Conn, ls *Source, userDN string) bool {
+	if len(ls.AdminFilter) > 0 {
```

Comment:
yes, that is current behavior but I can change it to not change admin flag if no `AdminFilter` is set